package.json

```diff
@@ -69,13 +69,16 @@
         "react-dom": "^17.0.0"
     },
     "devDependencies": {
-        "@exabyte-io/cove.js": "https://github.com/Exabyte-io/cove.js#3eb9647c8484b38a6166d804085cbec2a2dd96a5",
         "@exabyte-io/code.js": "2023.11.1-0",
-        "@exabyte-io/eslint-config": "^2022.11.17-0",
+        "@exabyte-io/cove.js": "https://github.com/Exabyte-io/cove.js#f1c074abf0ee5f162d01a645d7531bb13c793039",
```

Comment:
Is there a PR for cove??

Reply:
Nothing at https://github.com/Exabyte-io/cove.js/pulls right now - either create a PR or use published version

Reply:
It was a PR, but then got merged. I'll just bump the version